app/controllers/application_controller.rb

```diff
@@ -194,4 +194,16 @@ def analytics_exception_info(exception)
       exception_class: exception.class.name,
     }
   end
+
+  def validate_recaptcha
```

Comment:
I'm wondering why this logic belongs in the `ApplicationController` instead of as a concern or super class for the form objects. It seems to me that its really just another part of the form that needs to be validated, and this would simplify the implementation and the tests a bit since you won't have to alter the controllers and can test most of the functionality at the form object level instead of at the controller level.

Reply:
The form objects isn't a good option because verify_recaptcha is placed in ActionController::Base by the gem.  The reason why I chose ApplicationController over a concern is because I thought eventually Recaptcha would be part of most pages.

Reply:
I would think we would only need it for unauthenticated pages. Also, what does recaptcha provide that rate limiting with Rack Attack doesn't?

Reply:
Spinning through IPs in a block, spinning through anonymous IPs, etc.

Reply:
:+1: to just putting it where we need it for now. It isn't hard move it into the application controller later if we need to.

Reply:
Yes.  Adding it to a separate concern right now.